### PR TITLE
chore: Revert breaking changes on v3

### PR DESCRIPTION
**Pull request by gastonfournier**
## About the changes
Revert changes on v3 that are major changes. They're already in v4 branch

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ import (
 const (
 	deprecatedSuffix = "/features"
 	clientName       = "unleash-client-go"
-	clientVersion    = "4.0.0"
+	clientVersion    = "3.9.0"
```
Comment (kwasniew) [inline client.go]: should we do 3.9.1?

**Review by kwasniew — APPROVED** [review]
I'd only bump the patch version (especially if we leave the fix for dependent feature toggle evaluation)

File: client.go (hunk 1 above)
Comment (gastonfournier) [reply client.go]: I was going to do this later as part of a release, but let me double check how is this done

Comment (gastonfournier) [reply client.go]:
```suggestion
	clientVersion    = "3.9.1"
```

**Pull request merged**